Circular_kernel_2/include/CGAL/Circular_kernel_2/Intersection_traits.h

```diff
@@ -44,7 +44,8 @@ struct CK2_Intersection_traits<CK, typename CK::Circle_2, typename CK::Circle_2>
   typedef typename
   boost::variant< typename CK::Circle_2,
                   typename std::pair< typename CK::Circular_arc_point_2,
-                                      unsigned int > >
+                                      unsigned int >,
+                  int >
```

Comment:
There should be a comment with an explanation, or pointing to a message on Github explaining why that fixes anything.